.autopep8.py

```diff
@@ -10,7 +11,9 @@
 if path.startswith(current_path + '/project/'):
     path = path.replace(current_path + '/project/', '/src/')
     subprocess.run(['docker-compose', 'exec', '-T', 'web', 'sh', '-c',
-                    "PYTHONPATH=/src/extrasrc autopep8 {} {}".format(" ".join(sys.argv[1:-1]), path)],
+                    "PYTHONPATH=/src/extrasrc autopep8 {} {}".format(" ".join(
+                        sys.argv[1:-1]), path)],
```

Comment:
These lines were pretty clearly too long. Unless we have some >80 line standard different from what our linting suggests?

Reply:
Except to the .autopep8.py file itself apparently. There are several other python files with lines over 79 too so I am not sure why they are not caught.